package.json

```diff
@@ -36,6 +36,7 @@
     "dapp-chain": "GANACHE_ARGS='-b 2' concurrently -k -n ganache,dapp -p '[{time}][{name}]' 'yarn ganache:start' 'sleep 5 && yarn dapp'",
     "forwarder": "node ./development/static-server.js ./node_modules/@metamask/forwarder/dist/ --port 9010",
     "dapp-forwarder": "concurrently -k -n forwarder,dapp -p '[{time}][{name}]' 'yarn forwarder' 'yarn dapp'",
+    "test:attributions-check": "./development/attributions-check.sh",
```

Comment:
The `test:` prefix is used to run tests, but this isn't really a test, so it could be confusing.

What are your thoughts on grouping the two attributions-related scripts together? So something like `attributions:generate` and `attributions:check`.